python_files/data_preprocessor.py

```
@@ -0,0 +1,180 @@
+            'Channel',
+            'ID_Dest',
+            'Dest_State',
+            'Dest_City']
+                
+    cont_features = ['Amount', 'Old_Balance', 'New_Balance','Service_Charges']
+
+
+    df[cont_features] = scaler.transform(df[cont_features])
+
+    for col in objs:
+        df[col] = df[col].apply(lambda x: encoder_dict[col].get(x, -1))
```

Ruff (0.8.2)

Function definition does not bind loop variable `col`

(B023)